featuretools/entityset/serialization.py

```diff
@@ -1,175 +1,177 @@
-# -*- coding: utf-8 -*-
-
 import json
 import os
 import shutil
-import sys
-import uuid
-from tempfile import mkdtemp
 
-import numpy as np
 import pandas as pd
-from pandas.io.pickle import read_pickle as pd_read_pickle
-from pandas.io.pickle import to_pickle as pd_to_pickle
 
+from .. import variable_types
+
+TYPES = ['csv', 'pickle', 'parquet']
+SCHEMA = {
+    'entities':
+    ['id', 'index', 'time_index', 'variables', 'properties', 'loading_info'],
+    'relationships': {
+        'parent': ['parent_entity', 'parent_variable'],
+        'child': ['child_entity', 'child_variable']
+    }
+}
+VTYPES = {
+    getattr(variable_types, v)._dtype_repr: getattr(variable_types, v)
+    for v in dir(variable_types)
+    if hasattr(getattr(variable_types, v), '_dtype_repr')
+}
+
+
+def to_entity_descr(e):
```

Comment:
no need to abbreviate. just use `to_entity_description`

Reply:
Done